eth/abc.py

```diff
@@ -775,7 +775,7 @@ def __call__(self, computation: 'ComputationAPI') -> None:
     def as_opcode(cls: Type[T],
                   logic_fn: Callable[['ComputationAPI'], None],
                   mnemonic: str,
-                  gas_cost: int) -> Type[T]:
+                  gas_cost: int) -> T:
```

Comment:
Made same as:

https://github.com/ethereum/py-evm/blob/3f33ac8c2b7a190b187f971dc95005341ccd00ca/eth/vm/opcode.py#L43

Reply:
Yeah, I agree with your call here, too. The implementation is clearly returning an `Opcode` instance instead of class, and would require a *lot* of refactoring to change (if we even wanted to).

The only thing I'd add it that the ABC docs should update, too.